crates/transaction-emitter-lib/src/transaction_generator/publishing/raw_module_data.rs

```diff
@@ -18,10 +18,10 @@ use once_cell::sync::Lazy;
 pub static PACKAGE_METADATA_SIMPLE: Lazy<Vec<u8>> = Lazy::new(|| {
 	vec![
 		13, 71, 101, 110, 101, 114, 105, 99, 77, 111, 100, 117, 108, 101, 1, 0, 0, 0,
-		0, 0, 0, 0, 0, 64, 54, 50, 55, 65, 65, 66, 51, 68, 51, 55, 55, 70,
```

Comment:
Is this file auto generated on build?

Reply:
no, it's not autogenerated, you need to run this:

https://github.com/aptos-labs/aptos-core/pull/6104/files#diff-f7c951856576e8b55685aab56ef30d9c01c8791fb67ed675bddf42be5bd800e5